core/modules/system/system.module

```diff
@@ -947,34 +921,90 @@ function system_menu() {
     'file' => 'system.admin.inc',
     'weight' => 20,
   );
+
+  // URL handling.
+  $items['admin/config/urls'] = array(
+    'title' => 'URL handling',
+    'description' => 'Settings related to URLs.',
+    'position' => 'right',
+    'weight' => -20,
+    'page callback' => 'system_admin_menu_block_page',
+    'access arguments' => array('access administration pages'),
+    'file' => 'system.admin.inc',
+  );
+  $items['admin/config/urls/settings'] = array(
+    'title' => 'URL settings',
+    'description' => 'General URL-related configuration for your site.',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('system_urls_settings'),
+    'access arguments' => array('administer site configuration'),
+    'file' => 'system.admin.inc',
+  );
+  $items['admin/config/urls/settings/check'] = array(
+    'title' => 'Clean URL check',
+    'page callback' => 'backdrop_json_output',
+    'page arguments' => array(array('status' => TRUE)),
+    'access callback' => TRUE,
+    'type' => MENU_CALLBACK,
+    'file' => 'system.admin.inc',
+  );
+
+  // @deprecated: Remove backwards-compatibility redirect in 2.0.
+  $items['admin/config/search/urls'] = array(
+    'page callback' => 'backdrop_goto_deprecated',
+    'page arguments' => array('admin/config/urls/settings'),
+    'access arguments' => array('administer site configuration'),
+    'type' => MENU_CALLBACK,
+  );
+  // @deprecated: Remove backwards-compatibility redirect in 2.0.
+  $items['admin/config/search/clean-urls'] = array(
+    'page callback' => 'backdrop_goto_deprecated',
+    'page arguments' => array('admin/config/urls/settings'),
+    'access arguments' => array('administer site configuration'),
+    'type' => MENU_CALLBACK,
+  );
+  // @deprecated: Remove backwards-compatibility redirect in 2.0.
+  $items['admin/config/search/urls/check'] = array(
+    'page callback' => 'backdrop_goto_deprecated',
+    'page arguments' => array('admin/config/urls/settings/check'),
+    'access callback' => TRUE,
+    'type' => MENU_CALLBACK,
+  );
+
   // Additional categories
+  $items['admin/config/content'] = array(
+    'title' => 'Content authoring',
+    'description' => 'Settings related to formatting and authoring content.',
+    'position' => 'left',
+    'page callback' => 'system_admin_menu_block_page',
+    'access arguments' => array('access administration pages'),
+    'file' => 'system.admin.inc',
+  );
+  $items['admin/config/metadata'] = array(
+    'title' => 'Metadata',
+    'description' => 'Settings related to metadata.',
```

Comment:
Could we get a better description here? How about, "Settings related to meta tags that are commonly used by search engines."